### DISCUSSION TITLE
new pages are not moving to dist

### DISCUSSION THREAD

**Discussion by manideepcs024** [h5bp/html5-boilerplate#2507] [General]
It was working fine, but all of a sudden when added a new page. no page moved to dist. only old pages are there.

**Comment by roblarsen** [comment]
You'll need to provide more detail than that

**Reply by manideepcs024** [reply]
I added a new page, say addtocart.html or product.html.
and when I click on the url of that page from website, page did not loaded. and I checked dist folder there was no products page added to it.

**Reply by roblarsen** [reply]
I'm still not sure what you're doing. We don't have any documented behavior where pages automatically end up in `dist` if you add them somewhere else. 

If you add folders to `src` and then build the project the files will get copied over (the gulp task `copy:misc`) but that doesn't happen automatically. You have to build the project. That's not something that end users should be doing. Building the project using Gulp is basically for people developing HTML5 Boilerplate. 

If you're working in the `dist` folder itself, using Parcel, then the new file will just show up. That's one of the the expected ways for end users to build new sites.
